sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.cs

```diff
@@ -26,6 +26,7 @@ public Manager(LiveMetricsExporterOptions options, IPlatform platform)
 
             if (options.EnableLiveMetrics)
             {
+                InitializeMetrics(platform);
```

Comment:
Why to have 2 initialize calls here, can we call this from InitializeState

Reply:
This is because of the partial classes.
I could try to merge all the initialize bodies, but I got feedback on another PR that the use of certain variables was split across multiple files.